x-pack/plugins/reporting/export_types/common/lib/screenshots.js

```diff
@@ -206,10 +206,11 @@ export function screenshotsObservableFactory(server) {
       args: [layout.selectors.screenshot, { title: 'data-title', description: 'data-description' }],
       returnByValue: true,
     });
+
     return elementsPositionAndAttributes;
   };
 
-  const getScreenshots = async ({ browser, elementsPositionAndAttributes }) => {
+  const getScreenshots = async (browser, elementsPositionAndAttributes) => {
```

Comment:
I don't see a reason to remove the "named parameters" here? Usually it's easier to read the method calls that way, since you are not relying on any "arbitrary" ordering.